pkg/config/config.go

```diff
@@ -64,19 +64,32 @@ type Config struct {
 
 // TODO: https://github.com/jbrekelmans/kube-compose/issues/64
 // nolint
-func New() (*Config, error) {
-	fileName := "docker-compose.yml"
-	data, err := ioutil.ReadFile(fileName)
-	if err != nil {
+func New(file *string) (*Config, error) {
+	var data []byte
+	var err error
+	var fileName string
+	if file != nil {
+		data, err = ioutil.ReadFile(*file)
 		if os.IsNotExist(err) {
-			fileName = "docker-compose.yaml"
-			data, err = ioutil.ReadFile(fileName)
+			return nil, errors.Wrap(err, fmt.Sprintf("error loading docker-compose file at %#v", fileName))
```

Comment:
We can combine this if with the next one (e.g. in the next if we can still get something like "insufficient permissions" that needs to be wrapped with context)